sotodlib/mapmaking/demod_mapmaker.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, signals=[], noise_model=None, dtype=np.float32, verbose=False
         self.ncomp        = len(comps)
         self.singlestream = singlestream
 
-    def add_obs(self, id, obs, noise_model=None, split_labels=None):
+    def add_obs(self, id, obs, noise_model=None, split_labels=None, use_psd=True, wn_label = 'preprocess.noiseQ_mapmaking.white_noise'):
```

Comment:
Wrap that new arg onto next line. Remove spaces around "=".

I see this arg, as well as `use_psd`, aren't included in the docstring.  These args are documented elsewhere ... so either copy paste that here, or else reduce duplication and just refer the reader somehow ... "the use_psd and wn_label args are passed through to add_obs; see that docstring".